src/cmake/Modules/FindGMP.cmake

```diff
@@ -4,7 +4,7 @@ if (GMP_INCLUDE_DIR AND GMP_LIBRARIES)
 endif (GMP_INCLUDE_DIR AND GMP_LIBRARIES)
 
 find_path(GMP_INCLUDE_DIR NAMES gmp.h )
-find_library(GMP_LIBRARIES NAMES gmp libgmp REQUIRED)
+find_library(GMP_LIBRARIES NAMES gmp libgmp mpir REQUIRED)
 #find_library(GMPXX_LIBRARIES NAMES gmpxx libgmpxx )
```

Comment:
`mpir` is only needed on Windows, right?

Reply:
Yes, it's a replacement for gmp. It's available from vcpkg.

Reply:
Will it affect Linux or OSX machines that do not have `mpir`?

Reply:
No. It's an OR.  If gmp is installed, it is preferred.